x/authz/authorization_grant.go

```diff
@@ -51,10 +54,6 @@ func (g Grant) GetAuthorization() Authorization {
 }
 
 func (g Grant) ValidateBasic() error {
-	if g.Expiration.Unix() < time.Now().Unix() {
-		return sdkerrors.Wrap(ErrInvalidExpirationTime, "Time can't be in the past")
-	}
-
```

Comment:
This part is a **MUST** for 0.45.

Reply:
Related: https://github.com/cosmos/cosmos-sdk/security/advisories/GHSA-2p6r-37p9-89p2

Reply:
@dwedul-figure  0.45 is branched from 0.44, and the patch is already there. Here we take a slightly different approach.